corehq/apps/users/forms.py

```diff
@@ -750,17 +750,14 @@ def __init__(self, domain, attrs=None, id='supply-point', multiselect=False, que
 
     def render(self, name, value, attrs=None):
         location_ids = value.split(',') if value else []
-        from corehq.apps.locations.util import get_locations_from_ids
-        try:
-            locations = get_locations_from_ids(location_ids, self.domain)
-        except SQLLocation.DoesNotExist:
-            locations = []
+        locations = list(SQLLocation.active_objects
+                         .filter(domain=self.domain, location_id__in=location_ids))
         initial_data = [{'id': loc.location_id, 'name': loc.display_name} for loc in locations]
```

Comment:
nit picky but since you're only pulling these two fields you could use `values_list` to reduce the amount of data you need to pull from the DB